### PR TITLE
Adds styling to title font and removes unneeded submit button

### PR DESCRIPTION
**Pull request by attom2**
## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] Lint (`npm run lint`) has passed locally and any fixes were made for failures

## Pull request type

 

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## What is the new behavior?

- Adds pokemon font to the page, removes create-react-app leftover styling

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: src/Components/App/App.css [hunk 1]
```diff
@@ -2,37 +2,8 @@
...
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
-}
-
-.App-link {
-  color: #61dafb;
-}
```
Comment (attom2) [inline src/Components/App/App.css:29]: All old create-react-app styling

File: src/Components/Game/Game.css [hunk 2]
```diff
@@ -1,4 +1,6 @@
 .single-pokemon {
   height: 40vh;
   filter: brightness(0%);
-}
\ No newline at end of file
+  -webkit-user-drag: none;
```
Comment (attom2) [inline src/Components/Game/Game.css:4]: disallows users to drag darkened image away

File: src/Components/Game/Game.js [hunk 3]
```diff
@@ -30,15 +29,11 @@ const Game = ({pokemons}) => {
     }
     fetchSinglePokemon();
 
-  }, []);
+  }, [pokemons]);
```
Comment (attom2) [inline src/Components/Game/Game.js:32]: second parameter causes useEffect to update on change of pokemons

File: src/Components/Header/Header.css [hunk 4]
```diff
@@ -1,6 +1,17 @@
...
+  height: 20vh;
+  display: flex;
+  align-items: center;
+}
+.pokemon-title {
+  font-family: 'Pokemon Solid Normal';
+  font-size: 42px;
+  -webkit-text-stroke: 1px #2a75bb;
+  -webkit-text-fill-color: #ffcb05;
+  letter-spacing: 2px;
```
Comment (emontealvo) [inline src/Components/Header/Header.css:14]: Nice, you were able to import the font for the header. It looks slick!!

**Comment by MarchCorbin** [comment]
Awesome job getting the font working!

**Pull request merged**